daemons/controld/controld_globals.h

```diff
@@ -19,89 +19,94 @@
 #include <controld_fsa.h>       // enum crmd_fsa_state
 
 typedef struct {
-    // Booleans
-
-    //! Group of \p controld_flags values
+    // Group of \p controld_flags values
     uint32_t flags;
 
 
-    // Controller FSA
+    /* Controller finite state automata */
```

Comment:
pedantic: I think there's only one automaton per process, not worth changing unless we find something else

Reply:
thx